hexwall.qmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "TaskView Hexwall"
+format: 
+  dashboard:
```

Comment:
I guess we need to wait for the release of Quarto 1.4 then, because from https://quarto.org/docs/dashboards/layout.html I understand this is a pre-release feature only coming in the upcoming Quarto 1.4 release?